auto_submit/bin/server.dart

```diff
@@ -9,25 +9,27 @@ import 'package:auto_submit/helpers.dart';
 import 'package:shelf/shelf.dart';
 import 'package:shelf_router/shelf_router.dart';
 
-class AutosubmitServer {
-  AutosubmitServer();
+Future<Response> webhookHandler(Request request) async {
+  final String? reqHeader = jsonEncode(request.headers);
+  print('Header: $reqHeader');
+  return Response.ok(
+    jsonEncode(<String, String>{}),
```

Comment:
I would have this parrot back the response body to make it more testable, and drop the headers. There isn't much functionality beyond auth that comes from headers.